tests/Browser/Pages/Backend/Users/AdminDeleteUserTest.php

```diff
@@ -36,7 +36,7 @@ public function testVisitAdminUser()
      */
     public function testDeleteSuccess()
     {
-        factory(User::class, 5)->create();
+        factory(User::class, 4)->create();
```

Comment:
sao lại sửa chổ này

Reply:
chỗ này do lúc login em tạo 1 user để login rồi nên tạo 4 cái để nó test cho đúng anh khỏi sửa code ở dưới

Reply:
ko đúng chổ nào nhể?

Reply:
$this->assertSoftDeleted('users', ['id' => '4']); cái này nè anh đáng lẽ là $this->assertSoftDeleted('users', ['id' => '5']); mới đúng do em tạo thêm 1 user để đăng nhập đó anh